index.html

```diff
@@ -1077,6 +1173,32 @@ <h2><dfn>CaptureStartFocusBehavior</dfn>
         </div>
       </section>
 
+      <section>
+        <h2><dfn>DisplaySurfaceChangeCallback</dfn></h2>
+        <p>
+          Used to receive a new {{MediaStream}} if the [=display
+          surface=] associated with a [=capture-sessions=] changes and
+          a callback has been set on the associated
+          {{CaptureController/setDisplaySurfaceChangeCallback()}}
+        </p>
+        <div>
+          <pre class="idl"
+               >callback DisplaySurfaceChangeCallback = undefined
+               (MediaStream stream);</pre>
+          <section>
+            <h2>Callback <a class="idlType">DisplaySurfaceChangeCallback</a>
+              Parameters</h2>
+            <dl data-link-for="NavigatorUserMediaSuccessCallback" data-dfn-for=
+                "NavigatorUserMediaSuccessCallback" class="callback-members">
+              <dt>stream of type {{MediaStream}}</dt>
+              <dd>
+                The {{MediaStream}} object associated with the new
+                [=display surface=] that the user has selected.
```

Comment:
nit: I think the MediaStream might be a container and the tracks are the ones associated with the newly selected display surface.

Reply:
Updated wording and including mentioning the new tracks.